data/core.yaml

```
@@ -798,6 +798,9 @@ en:
       edtf: Add EDTF date
       edtf_label: Extended Date/Time Format
       edtf_placeholder: 1849~, 1804?, 189X, 1906/1908, 1814-23, 1960-05-01T13:00...
+    field_source: Add Source
```

If we make this a standard tooltip, it would be lowercase like the other ones:

```suggestion
    field_source: add source
```